apps/spruce/src/pages/version/taskDuration/TaskDurationTable.tsx

```diff
@@ -257,20 +274,26 @@ const getInitialParams = (queryParams: {
     initialFilters.push({ id: PatchTasksQueryParams.Variant, value: variant });
   }
 
+  let initialSort: SortingState = [];
+  if (sortBy && sortDir) {
+    initialSort = [
+      {
+        id: sortCategoryToColumnId[sortBy],
+        desc: sortDir === SortDirection.Desc,
+      },
+    ];
+  } else if (sorts) {
```

Comment:
Is it possible to just use `sorts` and remove the `sortBy`/`sortDir` parameters? Using just a single item in `sorts` would be a lot cleaner than having two separate cases for single vs multisort. We do this in the tests table for example

Reply:
I realized it would be a poor experience to temporarily utilize `sortDir`/`sortBy` so I added a casing in useTableSort to tell it when to only use 1 query parameter as an intermediate step before `sortDir` and `sortBy` is removed for in the rest of the application.